enterprise/backend/test/metabase_enterprise/semantic_search/core_test.clj

```diff
@@ -223,3 +235,23 @@
             (testing ":search-string is empty"
               (let [results (semantic.core/results (assoc search-ctx :search-string ""))]
                 (is (= fallback-results results))))))))))
+
+;; TODO (Chris, 2025-11-14) It's gross that we test this with mocking, but in-line with the rest of the test suite.
+;;      Ideally we would do more back box testing, and use actual embedding snapshots taken from a real model.
```

Comment:
Maybe that is more of a benchmark thing in the future and we see this more as testing the logic after whatever comes out of postgres.

Reply:
Yes that is fair 👍